Runtime/Scripts/DebugUtils/BillboardText.cs

```diff
@@ -0,0 +1,48 @@
+using TMPro;
+using UnityEngine;
+
+public class BillboardText : MonoBehaviour
+{
+
+    private Camera _mainCamera;
+    private TextMeshProUGUI _text;
+    private string _content;
+    private float _yOffset;
+
+    // Start is called before the first frame update
+    void Awake()
+    {
+        _mainCamera = Camera.main;
+        _text = GetComponentInChildren<TextMeshProUGUI>();
+        // Rotate the text 180 degrees, otherwise billboard will show it backwards
+        _text.transform.Rotate(Vector3.up, 180);
+        if (_content != null)
+        {
+            SetText(_content);
+            SetYOffset(_yOffset);
+        }
+    }
+
+    // Update is called once per frame
+    void LateUpdate()
+    {
+        transform.LookAt(_mainCamera.transform);

```

Comment:
This is slightly wrong (you can see the effect more prominently in the boss room video in the PR description - my billboard tilts too much). Any advice?

Reply:
maybe if you made the forward vector of billboard the camera's forward vector * -1?

Reply:
This problem is stuck in my head. I think this will work:

```
transform.LookAt(_mainCamera.transform);
transform.up = _mainCamera.up;
```

I think setting the `up` like that will keep your billboard looking correct regardless of the camera's rotation

Reply:
@RG-nAmKcAz's approach worked, took it from the existing sample in our SDK! Closing this comment